### PR TITLE
Fetch headlines with HTTPS

### PR DESCRIPTION
**Pull request by netzhuffle**
Plus better logging

### CODE REVIEW

File: 1_getHeadlines.js [hunk 1]
```diff
@@ -78,8 +78,9 @@ function getNextHeadline() {
 	}).on('error', error.bind(this, searchTerm));
 }
 
-http.get(config.frontPageURL, response => {
+https.get(config.frontPageURL, response => {
   let running = false;
+  console.info(`Fetching startpage. Status code ${response.statusCode}`);
```
Comment (dbrgn) [inline 1_getHeadlines.js]: Your indentation is a bit weird :)

**Review by dbrgn — COMMENTED** [review]
Why are there different libs for http and https? It's the same transport layer after all...

This is for fetching the content of the news pages, right? Do all of them support HTTPS? (20min for example does not...)

Does the library do certificate hostname verification?

File: 1_getHeadlines.js (hunk 1 above)
Comment (netzhuffle) [reply 1_getHeadlines.js]: Whoops, looks like I made a first step into the tabs+spaces mix hell. Time to ask autoformat for help, I guess …

**Comment by netzhuffle** [comment]
Crazy, right? Was very surprised when I saw that https://nodejs.org/api/http.html raises an exception with HTTPS, and even more surprised when I saw https://nodejs.org/api/https.html. At least both first-party libraries have the same API …

Yes, this script is for fetching the headlines. It’s mainly targeted for Blick am Abend, mostly because the other news pages forbid automatic fetching in their AGB and I had no interest in challenging it (but I can’t find that paragraph anymore on 20min). I guess deciding the right library dynamically would be the right way to go, I’ll fix that.

No idea on the last question.

**Review by dbrgn — APPROVED** [review]

**Review by rnestler — APPROVED** [review]

**Comment by rnestler** [comment]
@netzhuffle Is this ready to be merged?

**Pull request merged**